Gemfile

```diff
@@ -4,4 +4,5 @@ gem 'jekyll'
 gem 'github-pages'
 gem 'geocoder'
 gem 'lolize', :require => 'lolize/auto'
+gem 'jekyll-redirect-from'
```

Comment:
Is that already enough? I don't really know Jekyll, but I vaguely remember it also had a list of enabled plugins somewhere in its configuration 🤔 

May also be something from back in the days though...

Reply:
thanks for the comment! I looked through the config file, and it looks like this gem in already in there 🤔 (looks like it was me in fact who added it two years ago 😂 https://github.com/rails-girls-summer-of-code/summer-of-code/commit/92abe860bac41d2e88e61d7871cf3cff59c7d36a#diff-aeb42283af8ef8e9da40ededd3ae2ab2) so now I wonder if this even needs to be added to the gemfile, if it's a dependency of some sort as it seems like it is?

Reply:
the redirect seems to work without the gem added to the gemfile (used this to try: https://railsgirlssummerofcode.org/blog/2016-02-16-anniversary-campaign) but for the sake of following the instructions properly I'd leave it in there as well as in the config file, as that's what the gem docs say.

Reply:
Makes sense 👍 

I think it's already a dependency of Jekyll itself (https://github.com/jekyll/jekyll/blob/6a604daa5ac492817ab55be1f7bfe8bc3971e093/Gemfile) that's why it worked without explicitly having it in the Gemfile.

I btw. also don't see the pagination thing in the Gemfile 🙈 

Anyways, good to merge in my opinion ✅  Does the job.

Reply:
awesome, thanks for checking ❤️ 
Also: 🙈 thanks for noticing regarding the pagination thing, I'll take a look and potentially fix in a separate PR!